modules/microshift-custom-ca-cert-rotation.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-cert-rotation_{context}"]
+= Configuring `oc` to start with an expired custom certificates
+
+External server certificates are not automatically renewed. You must manually rotate your external certificates. However, you can configure the {oc-first} command-line tool to allow a start with an expired certificate. Then you can rotate the certificates as needed.
```

Comment:
Do you mean `oc` would start itself? I think this is a bit confusing.
Should we use `to allow execution with...` instead?

Reply:
We removed this module. I will delete reference to it completely. It does not apply here.